devito/passes/iet/mpi.py

```diff
@@ -130,7 +130,7 @@ def mpiize(iet, **kwargs):
         heb = user_heb if hs.is_Overlappable else sync_heb
         mapper[hs] = heb.make(hs)
     efuncs = sync_heb.efuncs + user_heb.efuncs
-    objs = sync_heb.objs + user_heb.objs
+    objs = filter_sorted(sync_heb.objs + user_heb.objs)
```

Comment:
does it need sorting? Or just unique

Reply:
I'm 99% sure just unique but sorting won't hurt